### DISCUSSION TITLE
import error: connection manager was closed

### DISCUSSION THREAD

**Discussion by Log-of-e** [knowledgefutures/pubpub#2740] [Support] [answered]
Hi,
 Just wanted to reach out, to pass on some error information.
Some imports error while polling for the workerTask:
From the browser network tab:

```
error :  "ConnectionManager.getConnection was called after the connection manager was closed!"
id:  "93732a91-8ae9-4f93-b09c-0d311a586d28"
isProcessing: false
output: null
```

**Comment by gabestein** [comment]
Thanks for reporting this. Do you mind linking us to the Pub you were trying to export, and which type of export, so we can try to replicate the error? It's related to a chance we made to better handle lots of export requests at the same time, but it's proven a bit difficult to track down because it seems to depend on a few conditions being present.

**Reply by tefkah** [reply]
~Hmm this could be due the upgrade, is this when running tests? I implemented some things to close open connections, but this should only affect tests. I sometimes saw this error but since the tests still passed I thought this was a superfluous error, but it might actually be relevant~

Nvm this is in the app

**Reply by tefkah** [reply]
This does seem like a regression, I can't seem to import anything. I can look into this today!

**Reply by Log-of-e** [reply]
This was for both browser and using API client from CLI
 I just made a pub and did 1 successful import using the browser.
 
 These are 2 pubs I had trouble importing to:
 https://testabcd123456789.pubpub.org/pub/qb4xei7s
 https://testabcd123456789.pubpub.org/pub/jq1u8736


 Trying just now using the CLI to create pub and import a file, resulted in the error at this pub: https://testabcd123456789.pubpub.org/pub/1vioa4un
 
 
 Thanks for looking into this. :)

**Reply by gabestein** [reply, marked as answer]
Thanks! We (I believe) found the cause and have a fix on its way up now.
